mne_bids/tests/test_mne_bids.py

```diff
@@ -195,6 +205,9 @@ def test_ctf():
     cmd = ['bids-validator', output_path]
     run_subprocess(cmd, shell=shell)
 
+    raw, events, _ = read_raw_bids(bids_basename + '_meg.ds',
+                                   output_path)
```

Comment:
This test is failing because the `.res4` file inside the CTF directory was not renamed correctly. Looks like we need a `copyfile_ctf` function as well. cc @sappelhoff

Reply:
also why didn't the validator pick this up? this should have been caught by the validator, no?

Reply:
okay I still can't get past this test because of the [empty file filler hack](https://github.com/mne-tools/mne-bids/blob/master/mne_bids/tests/test_mne_bids.py#L200-L207). If we comment out the hack, the reader works but the validator fails.